src/GovUk.Education.ExploreEducationStatistics.Admin/Services/Methodologies/MethodologyService.cs

```diff
@@ -197,6 +203,12 @@ await _context.Entry(methodology.ScheduledWithRelease)
         private Task<Either<ActionResult, Methodology>> UpdateMethodologyDetails(Methodology methodologyToUpdate,
             MethodologyUpdateRequest request)
         {
+            if (!request.IsDetailUpdateForMethodology(methodologyToUpdate))
+            {
+                // Details unchanged
+                return Task.FromResult(new Either<ActionResult, Methodology>(methodologyToUpdate));
```

Comment:
Similar to earlier, could async/await this method to avoid needing `Task.FromResult`.

Reply:
Made this change now, thanks 🙂